subsys/metrics/src/main/resources/default-metrics.conf

```diff
@@ -0,0 +1,8 @@
+[metrics]
```

Comment:
The paths you need:

* src/main/conf/conf.d/metrics.conf
* src/main/resources/default-metrics.conf

No, you'll still need the [metrics] header. The configuration uses `Include conf.d/*.conf` in the main.conf file for Indy, which causes the configuration reader to read those files and append their contents to the configuration content being parsed. So we still need section headers, even though that section is the only thing in the file.